greenback-kit-core/src/main/java/com/greenback/kit/model/AutoExport.java

```diff
@@ -0,0 +1,98 @@
+package com.greenback.kit.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class AutoExport extends GreenbackObject{
```

Comment:
Formatting - sometimes spaces on brackets and such.  Maybe intellij has a "format" method?